Styling/README.md

```diff
@@ -1344,6 +1347,66 @@ For example:
 
 **TODO : add note about GLSL implementations requires strict type comparisons among other things: https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/140**
 
+## Vector Data
+
+A [Vector Tile](../TileFormats/VectorData/README.md) is a collection of vector features such as points, polygons, and polylines. Vector features can be styled similar to other features with `color` and `show`. Points have several other styling options:
+
+The following style properties apply to geometry features in vector tiles:
+
+| Style | Type | Description | Default |
+| --- | --- | --- | --- |
+| `show` | `Boolean` | Whether or not to show the feature. | `true` |
+| `color` | `vec4` | The color of the feature. | `color('#FFFFFF')` |
+
+The following style properties apply to point features in vector tiles:
+
+| Style | Type | Description | Default |
+| --- | --- | --- | --- |
+| `show` | `Boolean` | Whether or not to show the feature. | `true` |
+| `color` | `vec4` | The color of the point. `color` is ignored when `image` is defined. | `color('#FFFFFF')` |
+| `pointSize` | `Number` | The size of the point in pixels. `pointSize` is ignored when `image` is defined. | `8.0` |
+| `pointOutlineColor` | `vec4` | The color of the point outline. `pointOutlineColor` is ignored when `image` is defined. | `color('#000000')` |
+| `pointOutlineWidth` | `Number` | The width, in pixels, of the point outline. `pointOutlineWidth` is ignored when `image` is defined. | `0.0` |
+| `labelText` | `String` | The text to display for the point. `'\n'` can be added to separate lines of text. | `undefined` |
+| `labelColor` | `vec4` | The color of the label's text. | `color('#FFFFFF')` |
+| `labelStyle` | `Number` | The label style: **fill** (`0`), **outline** (`1`), or **both** (`2`). See **Table1**.| `0` |
+| `labelOutlineColor` | `vec4` | The color of the text outline. | `color('#FFFFFF')` |
+| `labelOutlineWidth` | `Number` | The width of the text outline. | `1.0` |
+| `font` | `String` | The font of the displayed text. | `"30px sans-serif"` |
+| `backgroundColor` | `vec4` | The label background color. | `rgba(42, 42, 42, 0.8)` |
```

Comment:
All the other color defaults are specified as css color strings, this should probably as well. (Also in schema).